app/helpers/application_helper.rb

```diff
@@ -506,4 +506,27 @@ def app_metadata
   def ui_settings
     { perPage: Setting['entries_per_page'], perPageOptions: per_page_options }
   end
+
+  def apipie_dsl_example(source, output = nil)
+    text = content_tag(:p, _('Example input:')) +
+      content_tag(:pre, source)
+
+    if output.present?
+      text += content_tag(:p, _('Example output:')) +
+        content_tag(:pre, output)
+    end
+
+    text.html_safe
+  end
+
+  def apipie_erb_wrap(content, mode: :loud, open_trim: false, close_trim: false)
```

Comment:
apipie-dsl uses its own [helper](https://github.com/ofedoren/apipie-dsl/blob/master/app/helpers/apipie_dsl_helper.rb), which can be reopened and extended. Those methods are not currently visible in views.